app/helpers/routerHelper.js

```diff
@@ -5,12 +5,24 @@ export const withCatch = (logger, route) => async (req, res, next) => {
   try {
     return await route(req, res, next);
   } catch (err) {
+    let responseData;
+
     if (err instanceof ErrorContext) {
       return next(err);
     }
+
+    if (err.response) {
+      responseData = err.response.data;
+    }
```

Comment:
Can we do something like this?

```js
const responseData = err.response ? err.response.data : undefined;
```

Reply:
We can, the reason I have done it like this is because on other projects there is already a statement checking for err.response and I was just being consistent. I will update it here.